include/nodes/internal/FlowScene.hpp

```diff
@@ -106,33 +106,46 @@ class NODE_EDITOR_PUBLIC FlowScene
    * @brief Node has been created but not on the scene yet.
    * @see nodePlaced()
    */
-  void nodeCreated(Node &n);
+  void
+  nodeCreated(QtNodes::Node &n);
 
   /**
    * @brief Node has been added to the scene.
    * @details Connect to this signal if need a correct position of node.
    * @see nodeCreated()
    */
-  void nodePlaced(Node &n);
+  void
+  nodePlaced(QtNodes::Node &n);
 
-  void nodeDeleted(Node &n);
+  void
+  nodeDeleted(QtNodes::Node &n);
 
-  void connectionCreated(Connection const &c);
-  void connectionDeleted(Connection const &c);
+  void
+  connectionCreated(QtNodes::Connection const &c);
 
-  void nodeMoved(Node& n, const QPointF& newLocation);
+  void
+  connectionDeleted(QtNodes::Connection const &c);
 
-  void nodeDoubleClicked(Node& n);
+  void
+  nodeMoved(QtNodes::Node& n, const QPointF& newLocation);
 
-  void connectionHovered(Connection& c, QPoint screenPos);
+  void
+  nodeDoubleClicked(QtNodes::Node& n);
 
-  void nodeHovered(Node& n, QPoint screenPos);
+  void
+  connectionHovered(QtNodes::Connection& c, QPoint screenPos);
 
-  void connectionHoverLeft(Connection& c);
+  void
+  nodeHovered(QtNodes::Node& n, QPoint screenPos);
 
-  void nodeHoverLeft(Node& n);
+  void
+  connectionHoverLeft(QtNodes::Connection& c);
 
-  void nodeContextMenu(Node& n, const QPointF& pos);
+  void
+  nodeHoverLeft(QtNodes::Node& n);
+
+  void
+  nodeContextMenu(QtNodes::Node& n, const QPointF& pos);
```

Comment:
We must be consistent: do move the function name to the new line or keep it here (as below)?

Reply:
Just you choose. I guessed that was your coding style since I never seen it elsewhere. Anyway the whole project has mixed style so I don't know which one to follow. If I could choose I'd prefer not to spread function signature in muiltiple line for both headers and cpp, except for long parameters, but in the end, your code your rules :)

Reply:
The following files use a verbose style:

Connection.hpp, NodeDataModel.hpp, ConnectionGeometry.hpp, ConnectionGraphicsObject.hpp, Node.hpp, NodeGeometry.hpp, NodeGraphicsObject.hpp, NodeState.hpp, Style.hpp, StyleCollection.hpp

The following files use a standardy style:

ConnectionStyle.hpp DataModelRegistry.hpp FlowView.hpp FlowViewStyle.hpp, NodeData.hpp, NodeStyle.hpp, PortType.hpp

These use mixed style:
ConnectionState.hpp, FlowScene.hpp

Since there's no a uniform indentation style, I would leave this one as is, and then I would create another pull request with an unique indentation style (you will tell which one). A style cleanup would greatly benefit the project, especially for samples (I already done it there in my fork).

Reply:
good!